File: trunk/SecurityToken.vb
Public Class SecurityToken
    Private _Token As String
    Private _Username As String
    Private _Password As String
    Private _EncryptedPassword As String

    Public Property Token() As String
        Get
            Return _Token
        End Get
        Set(ByVal Value As String)
            _Token = Value
        End Set
    End Property

    Public Property Username() As String
        Get
            Return _Username
        End Get
        Set(ByVal Value As String)
            _Username = Value
        End Set
    End Property

    Public Property Password() As String
        Get
            Return _Password
        End Get
        Set(ByVal Value As String)
            _Password = Value
        End Set
    End Property
End Class

File: SecurityToken.cs
namespace BibleWebService
{
    public class SecurityToken
    {
        private string _Token;
        private string _Username;
        private string _Password;
        private string _EncryptedPassword;

        public string Token
        {
            get { return _Token; }
            set { _Token = value; }
        }

        public string Username
        {
            get { return _Username; }
            set { _Username = value; }
        }

        public string Password
        {
            get { return _Password; }
            set { _Password = value; }
        }
    }
}